resources/views/grid_screen_index.php

```diff
@@ -1,206 +1,217 @@
 <div class="grid-screen grid_screen-container">
-  <?php
+    <?php
     if (!empty($_GET['zype_parent'])) {
-      $parent_id=$_GET['zype_parent'];
+        $parent_id = $_GET['zype_parent'];
     }
 
-    $get_all = !empty($_GET['zype_get_all'])? $_GET['zype_get_all']: 0;
+    $get_all = !empty($_GET['zype_get_all']) ? $_GET['zype_get_all'] : 0;
 
     if (!empty($_GET['zype_str'])) {
-      $page = $_GET['zype_str'];
+        $page = $_GET['zype_str'];
     } else {
-      $page = 1;
+        $page = 1;
     }
 
-    if ($get_all!=0 and $get_all!=2) {
-      exit("can't load page");
+    if ($get_all != 0 and $get_all != 2) {
+        exit("can't load page");
     }
 
     $pagination = Config::get('zype.playlist_pagination', true);
 
-    $i=0; $j=0;
-  ?>
+    $i = 0;
+    $j = 0;
+    ?>
 
-  <div class="content-box grid_screen-box">
-    <?php if ((!empty($_GET['zype_items']) && $_GET['zype_items'] > 0 || $get_all == 2) && $parent_playlist ): ?>
-      <h2 class="playlist-view-title"><?php echo $parent_playlist->title ?></h2>
-    <?php endif ?>
+    <div class="content-box grid_screen-box">
```

Comment:
@Osoro Shouldn't we validate all the `$_GET['SOMETHING']` with `$this->request->validate` in `resources/controllers/Consumer/Gridscreen.php`?